marklogic-data-hub-central/ui/e2e/cypress/e2e/modeling/verifyConceptClassesInModeling.cy.tsx

```diff
@@ -365,4 +365,44 @@ describe("Concept classes in Modeling screen", () => {
     entityTypeTable.getConceptClass("TestConcept").should("not.exist");
   });
 
+  it("Create/Delete new relationship and new concept class from Table view", {defaultCommandTimeout: 120000}, () => {
+    cy.log("Add new concept class from table view");
+    cy.waitUntil(() => toolbar.getModelToolbarIcon()).click();
+    modelPage.selectView("table");
+    entityTypeTable.waitForTableToLoad();
+    cy.waitUntil(() => modelPage.getAddButton()).click();
+    modelPage.getAddConceptClassOption().should("be.visible").click({force: true});
+    conceptClassModal.newConceptClassName("TestConcept");
+    conceptClassModal.newConceptClassDescription("Test description.");
+
+    modelPage.openIconSelector("TestConcept");
+    modelPage.selectIcon("TestConcept", "FaAccessibleIcon");
+    modelPage.toggleColorSelector("TestConcept");
+    modelPage.selectColorFromPicker("#D5D3DD").click();
+    modelPage.toggleColorSelector("TestConcept");
+    if (Cypress.isBrowser("!firefox")) {
+      graphViewSidePanel.getConceptClassColor("TestConcept").should("have.css", "background", "rgb(213, 211, 221) none repeat scroll 0% 0% / auto padding-box border-box");
+    }
+    if (Cypress.isBrowser("firefox")) {
+      graphViewSidePanel.getConceptClassColor("TestConcept").should("have.css", "background-color", "rgb(213, 211, 221)");
+    }
+
+    modelPage.getIconSelected("TestConcept", "FaAccessibleIcon").should("exist");
+    conceptClassModal.getAddButton().click();
+    cy.waitForAsyncRequest();
+    conceptClassModal.getAddButton().should("not.exist");
+    cy.scrollTo("bottom");
+    table.scrollToFooter();
+    entityTypeTable.getConceptClass("TestConcept").should("exist").scrollIntoView();
```

Comment:
No need to tests all this part. The tests "Create/Edit and verify new concept class from Table view" is already doing all this checks.
Also I do not see the part were we create and edit a relationship related to a concept.